code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -419,7 +420,15 @@
 			new /obj/item/weapon/spellbook/oneuse/ice_barrage(get_turf(src))
 		if("alchemy")
 			new /obj/item/weapon/spellbook/oneuse/alchemy(get_turf(src))
-	new /obj/item/weapon/spellbook/oneuse/absorb(get_turf(src))
+	 
+	var/mob/U = src
```

Comment:
There's no need to cast this to `mob`, src is already a `/mob/living/carbon/human`.